tests/models/mamba/test_modeling_mamba.py

```diff
@@ -263,17 +275,24 @@ def prepare_config_and_inputs_for_common(self):
 )
 @require_torch
 class MambaModelTest(ModelTesterMixin, GenerationTesterMixin, PipelineTesterMixin, unittest.TestCase):
-    all_model_classes = (MambaModel, MambaForCausalLM) if is_torch_available() else ()
+    all_model_classes = (MambaModel, MambaForCausalLM, MambaForSequenceClassification) if is_torch_available() else ()
     all_generative_model_classes = (MambaForCausalLM,) if is_torch_available() else ()
     has_attentions = False  # Mamba does not support attentions
     fx_compatible = False  # FIXME let's try to support this @ArthurZucker
     test_torchscript = False  # FIXME let's try to support this @ArthurZucker
     test_missing_keys = False
-    test_model_parallel = False
     test_pruning = False
     test_head_masking = False  # Mamba does not have attention heads
+    test_model_parallel = False
+    test_mismatched_shapes = False  # MambaMixer follows a different initialization
```

Comment:
This seems a bit weird to me :thinking: Disabling the flag shouldn't be needed imo.

Could you add `get_input_embeddings` and `set_input_embeddings` methods for the ForSeqClassification class and see if it fixes those tests?